### DISCUSSION TITLE
Is it possible to dynamically toggle a plugin for a buffer?

### DISCUSSION THREAD

**Discussion by rbauduin** [LazyVim/LazyVim#5148] [Q&A] [answered]
I would like to manually toggle (possibly using a custom keymap) a plugin for a buffer. What's the best approach?

**Comment by folke** [comment, marked as answer]
that is something very general. What does toggling a plugin even mean?
That is different for each plugin, so no there's is no generic way to **toggle** a plugin.

**Reply by rbauduin** [reply]
Thanks for your clarification, this was not obvious to me.
My use case was to disable inline rendering by `3rd/image.nvim`, and in the meantime I saw they have a branch dedicated to this.
